### PR TITLE
Laststand Config / optimize loops

### PR DESCRIPTION
**Pull request by KamuiKody**
this should resolve future issues similar to #210

**Describe Pull request**
this pulls the continuous loop out checking if you are in a hospital bed and puts the loop into a function that is called when you are places in the bed and breaks it once you leave the bed

Also added a config option for the bleedout time when you are in last stand so people can easily adjust that time

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes] (Be honest)
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

**Review by sinphoenix — APPROVED** [review]
i tested and seems to work just fine. Myself nor others had errors

File: config.lua [hunk 1]
```diff
@@ -6,6 +6,7 @@ Config.WipeInventoryOnRespawn = true -- Enable or disable removing all the playe
 Config.Helicopter = "polmav" -- Helicopter model that players with the ambulance job can use
 Config.BillCost = 2000 -- Price that players are charged for using the hospital check-in system
 Config.DeathTime = 300 -- How long the timer is for players to bleed out completely and respawn at the hospital
+Config.DeathTime = 45 -- How long the timer is for players to bleed out completely and respawn at the hospital
```
Comment (charj) [inline config.lua:9]: We do not want to add this Duplicate config item.

**Review by charj — CHANGES_REQUESTED** [review]
See my above comments

File: config.lua (hunk 1 above)
Comment (KamuiKody) [reply config.lua:9]: Thank you for pointing this out not sure why i had a second variable but i will fix that tonight

File: client/laststand.lua [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Laststand = Laststand or {}
-Laststand.ReviveInterval = 360
+Laststand.ReviveInterval = Config.BleedoutInterval
```
Comment (charj) [inline client/laststand.lua:2]: I don't think we want this to use the same config item

File: config.lua [hunk 3]
```diff
@@ -32,7 +32,7 @@ Config.MajorArmoredBleedChance = 45 -- The chance, in percent, that a player wil
 Config.MaxInjuryChanceMulti = 3 -- How many times the HealthDamage value above can divide into damage taken before damage is forced to be applied
 Config.DamageMinorToMajor = 35 -- How much damage would have to be applied for a minor weapon to be considered a major damage event. Put this at 100 if you want to disable it
 Config.AlertShowInfo = 2 -- How many injuries a player must have before being alerted about them
-
+Config.BleedoutInterval = 50
```
Comment (charj) [inline config.lua:35]: A comment next to this config item wouldn't be a bad thing :)

Comment (KamuiKody) [reply config.lua]: fixed this! this was actually vital for this note i myself couldnt remember what variable to change to change this interval haha

File: client/wounding.lua [hunk 4]
```diff
@@ -152,6 +162,7 @@ CreateThread(function()
 	end
 end)
 
+
```
Comment (TheNeo2k) [inline client/wounding.lua:165]: extra whitespace? not that it matters, but its still a change.

**Pull request closed without merging**